deployment/config.go

```diff
@@ -93,7 +93,8 @@ type Config struct {
 	Report                report.Config
 	// Directory under which the .terraform directory and state files are managed.
 	// It will be created if it does not exist
-	TerraformStateDir string `default:"/var/lib/mattermost-load-test-ng" validate:"notempty"`
+	// ./ltstate is the default value used when `deployer.json`` is configured at all
+	TerraformStateDir string `default:"./ltstate" validate:"notempty"`
```

Comment:
I think this comment is not needed, since the tag is clear in my opinion:
```suggestion
	TerraformStateDir string `default:"./ltstate" validate:"notempty"`
```